packages/ui/src/server/controllers/strawberry-api.ts

```diff
@@ -32,29 +32,38 @@ async function chytProxyApiImpl(req: Request, res: Response) {
         'get_speclet',
     ]);
 
+    if (!engine) {
```

Comment:
It is better to check such parameters for specific allowed values
```ts
const ENGINES = {
    chyt: 'chyt_controller_base_url',
    spyt: 'livy_controller_base_url',
}
```
Then you don't need `isChyt` variable.

Reply:
ENGINES is not a suitable name. I did it a little differently